test/compilable/compile1.d

```diff
@@ -780,22 +780,6 @@ struct S13280
 }
 
 /***************************************************/
-// 13481
```

Comment:
I deleted this test instead of importing `std.algorithm`, because bug 13481 is specific to built-in attributes, and now that `.sort` is gone, the bug no longer applies.